merge_requests.go

```diff
@@ -107,6 +107,7 @@ type MergeRequest struct {
 	HasConflicts                bool `json:"has_conflicts"`
 	BlockingDiscussionsResolved bool `json:"blocking_discussions_resolved"`
 	Overflow                    bool `json:"overflow"`
+	FirstContribution           bool `json:"first_contribution"`
```

Comment:
Please place this above `TaskCompletionStatus` following the ordering in the docs

Reply:
done